README.md

```diff
@@ -63,7 +63,7 @@ Add this to your configuration:
 custom_updater:
   component_urls:
 # Dev build (unstable)
-    - https://raw.githubusercontent.com/keatontaylor/alexa_media_player/next/custom_components.json
+    - https://raw.githubusercontent.com/keatontaylor/alexa_media_player/dev/custom_components.json
```

Comment:
Because of #157, which fixes #159, your PR should simply delete lines 65 & 66. (I'm sorry for forgetting to update README.md in #157)